src/DefinitionGenerator.ts

```diff
@@ -116,6 +124,28 @@ export class DefinitionGenerator {
     return cleanedSchema;
   }
 
+  /**
+   * Walks through the schema object recursively and updates references to point to openapi's components
+   * @param schema JSON Schema Object
+   */
+  private updateReferences (schema: JSONSchema7): JSONSchema7 {
```

Comment:
Should this var have a version number in it? Shouldn't it just be a generic word like "schema"? As the schema version is not relevant to this function.